code_challenges.rb

```diff
@@ -21,6 +41,27 @@
 # Expected output: ['tea', 'soda water']
 
 
+# There must be a method that will be able to use the includ method but extract the actual value that is being held in the array instead of just returning a boolean value.
+# must determine how I will determine which letter is being called upon to see if the character is present in the element that is being inspected.
+
+
+
+
+
+def letter_scanner_o array
+  array.select {|item| item.include?('o')}
+  # array.select {|item| item.include?('o') || item.include?('a')}
+  # I was trying to create a method that would return two arrays each looking for a different letters
+  # only solution that came to mind would be to do two seperate methods
+end
+
+p letter_scanner_o beverages_array
+
+def letter_scanner_a array
+  array.select {|item| item.include?('a')}
+end
```

Comment:
I like these solutions but we can make it even more dynamic! we want to be able to enter _any_ letter and see if it's in one or more of the words in the array. to do that, we will need to be able to also send a letter as an argument. we can then utilize that letter in the `select`:
```
def letter_scanner(array, letter)
  array.select {|item| item.include?(letter)}
end

p letter_scanner_a(beverages_array, 'o')
```